data/presets/presets/leisure/fishing.json

```diff
@@ -0,0 +1,18 @@
+{
+    "icon": "aquarium",
```

Comment:
I am not certain this is the best icon here, perhaps a new icon needs to be created.